src/components/Divider/Divider.test.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Divider } from '.'
+import { SplitTextComponent } from './Divider.mocks'
+import { Type } from './Divider.types'
+import { render } from '../../test-utils'
+
+const { Vertical } = Type
+
+describe('Divider Component', () => {
```

Comment:
I'm not sure what is displayed in the snapshots in those cases (so in some of them the tests may not be effective), but there are some missing use cases related to `isDashed`, `isPlain`, `orientation`, and `orientationMargin`

Reply:
as in RTL we cannot test anything valuable for Divider component.
snapshots are used to test classnames

i'm adding missing tests (Y)

Reply:
As I said, it may not be necessary to add snapshot-based tests because if nothing valuable is displayed we are not testing anything. Take a look at what is shown in the various cases and judge for yourself whether the test is meaningful or not 😉

Reply:
snapshots for `orientation` and `orientationMaring` are not valuable at all as nothing is changing

Reply:
just added missing tests, i'm keeping snapshots as i consider useful testing the prop drilling via `classname` to the ant component. better than having no tests imho

Reply:
I agree with @danielemarostica